### PR TITLE
Update durable-data.rst

### PR DESCRIPTION
**Pull request by mellbye-Sesam**
Clean Important msg. 
More consistent use of high-durability disk and clarify the use case for durable data.

### CODE REVIEW

File: features/durable-data.rst [hunk 1]
```diff
@@ -3,23 +3,23 @@
...
+By enabling durable data, Sesam will store the pipe data in three replicas on a high-durability disk. This greatly reduces the likelihood of data loss.
 
 
 Use case
 --------
 
-For cloud subscriptions, data is backed up to an external service once every 24 hours. During a disaster recovery data written the last 24 hours can be lost. This might not be a critical problem when Sesam is pulling data from sources, as the data that was lost can be pulled again. For pipes with :ref:`HTTP endpoint source <http_endpoint_source>` and :doc:`non-idempotent <../idempotency>` sinks, this will most likely be a problem.
+For cloud subscriptions, data is backed up to an external service once every 24 hours. During a disaster recovery data written the last 24 hours can be lost. By using replicas and a high-durability disk for storage, the risk of data loss is greatly reduces. The risk of data loss might not be a critical problem when Sesam is pulling data from sources, as the data that was lost can be pulled again. However for pipes with :ref:`HTTP endpoint source <http_endpoint_source>` and :doc:`non-idempotent <../idempotency>` sinks, this will most likely be a problem.
```
Comment (grove) [inline features/durable-data.rst:12]: Typo: "reduces" should be "reduced"

File: features/durable-data.rst [hunk 2]
```diff
@@ -3,23 +3,23 @@
 Durable Data
 ============
 
-In our cloud subscriptions you now have the possibility to request that a pipes data is stored in three replicas. This reduces the likelihood of data loss.
+By enabling durable data, Sesam will store the pipe data in three replicas on a high-durability disk. This greatly reduces the likelihood of data loss.
```
Comment (grove) [inline features/durable-data.rst:6]: The "three replicas" part is sort of an implementation detail and I'm not sure we have to mention that at all.

**Review by grove — CHANGES_REQUESTED** [review]
I've added a few comments.

**Review by djalil-bouhachi — APPROVED** [review]

File: features/durable-data.rst (hunk 2 above)
Comment (mellbye-Sesam) [reply features/durable-data.rst:6]: agreed, will fix

**Review by djalil-bouhachi — APPROVED** [review]

**Review by grove — APPROVED** [review]

**Pull request merged**